### ISSUE TITLE
Not possible to select multiple job status in drop down

### ISSUE THREAD

**Issue by braultatgithub** [redhat-cip/dci-ui#80]
- It would be nice to be able to select multiple job status in the job status drop down.
- It would be nice to have an extra filter based on component or topic. (Right now only Job status and Team) are offered.
See screenshot
![screen shot 2018-11-19 at 2 31 22 pm](https://user-images.githubusercontent.com/20091420/48714320-c2689380-ec12-11e8-8a8f-03c5a7ca874c.png)

**Comment by guillaumevincent** [comment]
This one is blocked by https://github.com/redhat-cip/dci-control-server/issues/76

**Comment by guillaumevincent** [comment]
I close this one. I will update the issue on dic-control-server
The issue is in the backlog with low priority
